taskcluster/.shared.yml

```diff
@@ -7,3 +7,8 @@ nodejs:
     prep_4: 'echo "deb http://deb.nodesource.com/node_4.x trusty main" > /etc/apt/sources.list.d/nodesource.list && wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -'
     prep_5: 'echo "deb http://deb.nodesource.com/node_5.x trusty main" > /etc/apt/sources.list.d/nodesource.list && wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -'
     prep_6: 'echo "deb http://deb.nodesource.com/node_6.x trusty main" > /etc/apt/sources.list.d/nodesource.list && wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -'
+system:
+  username: 'build-user'
+  homedir:
+    linux: '/home/build-user'
+    osx: '/Users/build-user'
```

Comment:
The way this is done, you just added an additional layer of abstraction to every place that used to have the homedir hardcoded. Can't you define this variable as simply `homedir` on the Darwin and Linux bases, so that you can refer to them as `system.homedir` everywhere? If not, I don't understand what this is improving.

Reply:
But then we would need to redefine that variable at several places: in `.build.yml`, in darwin, linux and test-linux bases. The goal here was to have both path centralized, and avoid hardcoding too many values.